hist/hist/inc/TH1.h

```diff
@@ -200,8 +200,12 @@ class TH1 : public TNamed, public TAttLine, public TAttFill, public TAttMarker {
    virtual Bool_t   Add(TF1 *h1, Double_t c1=1, Option_t *option="");
    virtual Bool_t   Add(const TH1 *h1, Double_t c1=1);
    virtual Bool_t   Add(const TH1 *h, const TH1 *h2, Double_t c1=1, Double_t c2=1);
-   virtual void     AddBinContent(Int_t bin);
-   virtual void     AddBinContent(Int_t bin, Double_t w);
+   /// Increment bin content by 1.
+   /// Passing an out-of-range bin leads to undefined behavior
+   virtual void     AddBinContent(Int_t bin) = 0;
+   /// Increment bin content by a weight w.
+   /// Passing an out-of-range bin leads to undefined behavior
```

Comment:
That is too strong of a statement.  For many derived classes, the behavior is specified (will increase the overflow/underflow bins).

Reply:
Nevermind, this is the function taking a bin number not a value.